amzn/rheoceros
# Rebase: v2023.09.27

Rebase onto internal repo.

- new batch compute drivers(SM, EMR)
- new signal types (SNS) that can be used as standalone signals or linked to S3 data
- enhancements in diagnostics (e.g custom dashboards)
- custom platform extensions (e.g DynamoDBTable). Multiple 'Extension' s can be implemented to extend RheocerOS platform model with custom cloud resources
- more examples covering both existing and new features
- major/minor bug fixes.

Testing:

- run internal testing suite (unit/integ tests)
- run unit-tests after rebase and modifications
   - local FS permission is required for 1 test (ignored) to create a temporary symlink for incompatibility check.
